apps/expo/src/app/muscleGroup/index.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useState } from 'react'
+import { Dimensions, Image, Text, View, StyleSheet } from 'react-native'
+import { router } from 'expo-router'
+import tailwind from '@/tooling/tailwind'
+import { Ionicons, MaterialCommunityIcons } from '@expo/vector-icons'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import Body from 'react-native-body-highlighter'
+import ToggleSwitch from '~/components/toggle/Toggle'
+import FrontBackSwitch from '~/components/muscleGroup/FrontBackSwitch'
+
+export type muscleSelect =
+  | "abs"
+  | "adductors"
+  | "ankles"
+  | "biceps"
+  | "calves"
+  | "chest"
+  | "deltoids"
+  | "deltoids"
```

Comment:
there are duplicate "deltoids" fields here

Reply:
Thanks for caching it😉